src/pages/communication-page/communication.ts

```diff
@@ -113,6 +115,9 @@ export class CommunicationPage extends PracticeableBasePageComponent {
       }
     }
 
+    // Will be required to cater for more than Cat B at a later stage
+    this.store$.dispatch(new PopulateTestCategory(TestCategory.B));
```

Comment:
You should do this in `journal.effects.ts` where all the test population occurs.